python/pyspark/sql/readwriter.py

```diff
@@ -282,6 +282,23 @@ def parquet(self, *paths):
         """
         return self._df(self._jreader.parquet(_to_seq(self._spark._sc, paths)))
 
+    @since(2.2)
+    def parquet(self, path):
```

Comment:
Thanks @holdenk , I learned a new thing of python. I reverted the changes on parquet, It would be very weird to change it as `def parquet(self, *paths, path=None):` and  `def parquet(self, **kwargs:)` would break the code without using keyword argument, e.g. `parquet("p_file")`